seed_services_cli/stage_based_messaging.py

```diff
@@ -167,6 +167,37 @@ def messages_import(ctx, csv, json):
             api.create_message(msg)
 
 
+@click.option(
+    '--csv', type=click.File(file_open_mode),
+    help=('CSV file with columns for the endpoint'))
+@click.option(
+    '--json', type=click.File(file_open_mode),
+    help=('JSON objects, one per line for the endpoint'))
+@click.pass_context
+def messages_update(ctx, csv, json):
+    """ Send message updates to the Stage Based Messaging service.
+        binary_content fields should refer to filename in the current folder
+    """
+    if not any((csv, json)):
+        raise click.UsageError("please specify either --csv or --json")
+    api = get_api_client(ctx.obj.stage_based_messaging.api_url,
+                         ctx.obj.stage_based_messaging.token)
+    if csv:
+        for msg in messages_from_csv(csv):
+            if msg["binary_content"] is not None and \
+                    msg["binary_content"] != "":
+                msg = create_binarycontent(api, msg)
+            click.echo("Updating message to messageset %(messageset)s." % msg)
+            api.update_message(msg)
+    if json:
+        for msg in messages_from_json(json):
+            if msg["binary_content"] is not None and \
+                    msg["binary_content"] != "":
+                msg = create_binarycontent(api, msg)
+            click.echo("Updating message to messageset %(messageset)s." % msg)
+            api.update_message(msg)
```

Comment:
This seems to be some repeated code. If the data returned from `messages_from_csv` and `message_from_json` is in the same format, maybe we can do something like:
```
if csv:
    messages = messages_from_csv(csv)
elif json:
    messages = messages_from_json(json)

for message in messages:
    ....
```

Reply:
I agree. I will call the messages msgs though, as there already is a messages(...) function defined.

Reply:
👍 Variable names can be whatever works best, it was just a guideline to show the general code structure that I was suggesting :)